sciolyid/cogs/media.py

```diff
@@ -85,7 +82,7 @@ def __init__(self, bot):
     # help text
     @commands.command(
         help="- Sends a random image for you to ID",
-        aliases=["p", config.options["id_type"], config.options["id_type"][0]],
+        aliases=["p", config.options["id_type"][:-1], config.options["id_type"][0]],
```

Comment:
also having the `config.options["id_type"][0]` may conflict with other command aliases